src/libs/ajax.js

```diff
@@ -76,12 +79,24 @@ const withRetryOnError = _.curry(wrappedFetch => async (...args) => {
   return wrappedFetch(...args)
 })
 
+
+// Captures given course in error message, compares to root addresses in dav.json plus https://storage.googleapis.com/,
```

Comment:
```suggestion
// Captures given course in error message, compares to root addresses in app configuration plus https://storage.googleapis.com/,
```

Spelling: "dav" vs "dev". Also, this isn't necessarily dev.json, but whatever configuration is specified by the environment.